js/left_nav_tool_visibility_indicator.js

```diff
@@ -0,0 +1,36 @@
+// source: TLT Canvas Liaison FAQ
+var huToolVisibilityRestricted = [
+  // Course-level
+  "Manage People",
+  "Manage Sections",
+  "Course Emailer"
+];
+
+function huFuzzyVisUpdate(tool){
+  return $("a[class^='context_external_tool']:contains('"+ tool + "')")
```

Comment:
Can we make these selectors a little more selective just to be sure we are only targeting the nav items?